src/styles/hamburgerInputPadding/index.android.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Android platform supports the "hamburgerInputPadding" property, so this
+ * can be added to the object
```

Comment:
Please include a link to the react native issue you found in this code so we can understand why the decision was made to do this.